### ISSUE TITLE
Fetch deviceName for current capability

### ISSUE THREAD

**Issue by mindmelting** [angular/protractor#1732]
Hello

I need to have a specific configuration set for each device in my multiCapabilities, I would like to reference this configuration either by deviceName or within the capability object. How would I go about fetching this in my test or in onPrepare?

Thanks

**Comment by juliemr** [comment]
Will the change pending at https://github.com/angular/protractor/pull/1724 solve this for you?

**Comment by juliemr** [comment]
Let me rephrase that to - I believe that #1724 will resolve this for you - it will be in the next release, and is now in master. Use with `browser.getProcessedConfig().then(function(config) { config.capabilities.deviceName... });`

Closing this issue since it will be resolved in the next release!

**Comment by mindmelting** [comment]
Amazing, thanks!
